api/task_manager.cc

```diff
@@ -99,7 +98,6 @@ future<full_task_status> retrieve_status(const tasks::task_manager::foreign_task
     s.type = task->type();
     s.parent_id = task->get_parent_id();
     s.abortable = task->is_abortable();
-    s.module = task->get_module_name();
```

Comment:
Can be sent separately

Reply:
I would leave it as is for the sake of backports

Reply:
But that's exactly where it could interfere (in theory) with backports.
Since it's orthogonal and doesn't fix anything, it's not really required for backport, hence we prefer to reduce the churn.